hoomd/hpmc/update.py

```diff
@@ -519,38 +518,38 @@ class Shape(Updater):
     Args:
         trigger (Trigger): Call the updater on triggered time steps.
 
-        step_size (float): Default maximum size of shape trial moves.
+        shape_move (ShapeMove): Type of shape move to apply when updating shape
+            definitions
+
+        step_size (float, **default**: 0.2): Default maximum size of shape trial
+            moves.
 
-        pretend (bool, optional): When True the updater will not actually update
-            the shape definitions. Instead, moves will be proposed and the
-            acceptance statistics will be updated correctly.
+        pretend (bool, **default**: `False`): When True the updater will not
```

Comment:
For attributes we usually use the `optional` flag, and mention the default `(**default**: False)` at the end of the description.